README.md

```diff
@@ -49,6 +39,10 @@ https://github.com/KillianLucas/open-interpreter/assets/63927363/37152071-680d-4
 
 ```shell
 pip install open-interpreter
+
+# ubuntu 22.04.3 LTS
+# it is recommended to create a virtual environment 
+sudo pip3 install open-interpreter 
```

Comment:
Using sudo to install pip is not the recommended best practice.
Most users don't have pip3 as the primary alias for pip install.